src/ejercicios.js

```diff
@@ -1 +1,25 @@
 // @TODO Completar...
+
+//1
+const contenedor = document.getElementById("contenedor");
+
+//2
+const lista = document.createDocumentFragment();
+
+//3
+const ul = document.createElement("ul");
+
+lista.appendChild(ul);
+
+//5
+const template = document.getElementById("mitemplate").content;
+
+for (i = 0; i < 50; i++) {
```

Comment:
Declarar variable `i`.
```suggestion
for (let i = 0; i < 50; i++) {
```